server_test.go

```diff
@@ -313,16 +313,20 @@ func TestServer(t *testing.T) {
 }
 
 func newServer(handler redis.Handler) (srv *redis.Server, url string) {
+	return newServerTimeout(handler, 100)
+}
+
+func newServerTimeout(handler redis.Handler, timeout time.Duration) (srv *redis.Server, url string) {
 	l, err := net.Listen("tcp", "127.0.0.1:0")
 	if err != nil {
 		panic(err)
 	}
 
 	srv = &redis.Server{
 		Handler:      handler,
-		ReadTimeout:  100 * time.Millisecond,
-		WriteTimeout: 100 * time.Millisecond,
-		IdleTimeout:  100 * time.Millisecond,
+		ReadTimeout:  timeout * time.Millisecond,
+		WriteTimeout: timeout * time.Millisecond,
+		IdleTimeout:  timeout * time.Millisecond,
```

Comment:
Sure. Sorry, I cargo-culted it from the previous implementation. Will RTFM and do it properly.

Thanks!